### ISSUE TITLE
Added Card via HACS

### ISSUE THREAD

**Issue by meremortals70** [Deejayfool/hass-shutter-card#20]
I added the Shutter Card via HACS and receive and error _Custom element doesn't exist: shutter-card._

Should I just add it manually?

**Comment by Yonny24** [comment]
I had to restart HA several time to get it to show. Had the same issue. Try adding it manually otherwise.

**Comment by Deejayfool** [comment]
The ressource should normally be added automatically by HACS. If it's not the case, then you have to add it manually as @Yonny24  said.

**Comment by meremortals70** [comment]
> The ressource should normally be added automatically by HACS. If it's not the case, then you have to add it manually as @Yonny24  said.

Thanks but that was the first thing I checked.

**Comment by MaxWinterstein** [comment]
Same issue here. Added manually via Lovelace resources.

Never had that issue before, restart did not solve that problem.

**Comment by srxvx** [comment]
Has someone figured out a solution? I added it via HACS and restarted multiple times, as well as doing it manually.
I still get "Custom element doesn't exist: shutter-card" error.

**Comment by svente08** [comment]
Same error - not working via HACS nor manually. Someone fixed it?

**Comment by meremortals70** [comment]
Yes I got it working but removed it again because it used to much screen real estate

**Comment by svente08** [comment]
Would you share the fix please?

**Comment by meremortals70** [comment]
I'm sorry I can't remember. I can tell you where it is installed etc, let me know if that would help

**Comment by svente08** [comment]
The js file is in place, copied it to different places, linked it in lovelace-ui, service real reboots, many installs/deinstalls with HACS but the UI doesn't recognise...

**Comment by meremortals70** [comment]
![image](https://user-images.githubusercontent.com/52735978/100856714-7fb34480-34d7-11eb-9a02-dae39029e462.jpeg)

My resources entry

![image](https://user-images.githubusercontent.com/52735978/100856734-8772e900-34d7-11eb-8125-958107369e4f.jpeg)

Physical location

**Comment by svente08** [comment]
Same on my end... Wired...

**Comment by meremortals70** [comment]
I just added a card, and it worked.

![image](https://user-images.githubusercontent.com/52735978/100858660-00734000-34da-11eb-9adc-1405fb2529b1.jpeg)

**Comment by svente08** [comment]
I found the issue. I configured month ago an extra yaml for resources. Sorry my fault.

But  what I see now "invert_percentage" inverts the % AND the shutter direction. Can you confirm?

**Comment by nikandgr** [comment]
Hi there 
I have  installed shutter card using HACS. Resourses seems ok. I added a custom card and i can not use the animation to move up or down my roller whilist the buttons seem to work. Also can not show the percentage (undefined%) although i have set upper and lower limmits on my roller and i can see it in tuya app. 
Can someone help ? 
![Untitled0](https://user-images.githubusercontent.com/37958375/104956548-057e4700-59d5-11eb-9d79-b993154e5899.png)
